mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -231,6 +264,9 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD a isInFluid (Lzc;Z)Z
```

Comment:
While we are at it, this could be renamed to `isSubmergedIn`. It would be make it more explicit that it checks for whether the entity has said fluid up to their eyes.

Reply:
* ditto `isSubmerged`